### DISCUSSION TITLE
quarkus: commant not found

### DISCUSSION THREAD

**Discussion by goverdhan07** [quarkusio/quarkus#27693] [Q&A]
Im setting up quarkus in my local, following the steps from [here](https://quarkus.io/get-started/) trying via `sdkman` getting below error 
`zsh: command not found: quarkus`
 Please let me know if I'm missing something

**Comment by BrkSe** [comment]
Hello. I encountered the same error. I tried a different step.

`brew install quarkusio/tap/quarkus`

This step worked.
